spec/models/task_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe Task do
+  include ::Spec::Support::TenantIdentity
+
```

Comment:
@hsong-rh If we used safe_mode https://github.com/travisjeffery/timecop#timecopsafe_mode we might not need the after since all our Timecop calls are in a block